client/my-sites/themes/theme-showcase.jsx

```diff
@@ -159,7 +159,10 @@ class ThemeShowcase extends Component {
 
 	scrollToSearchInput = () => {
 		if ( ! this.props.loggedOutComponent && this.scrollRef && this.scrollRef.current ) {
+			const yOffset = -55;
```

Comment:
That'll most certainly need a comment so we can know where it came from.

Reply:
Comment added.